benchmarks/CMakeLists.txt

```diff
@@ -28,6 +28,11 @@ blt_add_executable(
   SOURCES copy_stress_test.cpp
   DEPENDS_ON ${stress_test_depends})
 
+blt_add_executable(
+  NAME fixed_pool_stress_test
+  SOURCES fixed_pool_stress_test.cpp
+  DEPENDS_ON ${stress_test_depends})
+
```

Comment:
I realize that this is simply using the current pattern "_stress_test" for the name, but I wonder whether the name would more accurately be called something like "benchmark_fixed_pool"?  I am thinking about what people might look for when the do an `ls ./bin` and see all of the executables there.  It might be nice to see all of the benchmarks, versus unit tests, versus stress tests, etc..

Thoughts?

Reply:
I think we should definitely discuss a naming convention here because I realized that, after not looking at this since like last wednesday, I at first thought this was a "fixed" version of the pool_stress_test.cpp. So I do think it could be called something better that would give a bit more clarity as to what it is/measures/etc. Perhaps this is something to discuss for tomorrow!

Reply:
As discussed, we can probably do this at a later time.  I suggest that a separate Jira ticket be written to describe the naming that should be used for all of the executables placed in the ./bin directory.

Reply:
Sounds good!

Reply:
See UM-849